solr/core/src/test/org/apache/solr/security/BasicAuthIntegrationTest.java

```diff
@@ -295,12 +295,12 @@ public void testBasicAuth() throws Exception {
       verifySecurityStatus(cl, baseUrl + "/admin/info/key", "key", NOT_NULL_PREDICATE, 20);
       assertAuthMetricsMinimums(17, 8, 8, 1, 0, 0);
 
-      String[] toolArgs = new String[] {"status", "--solr-url", baseUrl};
+      String[] toolArgs =
+          new String[] {"status", "--solr-url", baseUrl, "--credentials", "harry:HarryIsUberCool"};
       ByteArrayOutputStream baos = new ByteArrayOutputStream();
       PrintStream stdoutSim = new PrintStream(baos, true, StandardCharsets.UTF_8.name());
       StatusTool tool = new StatusTool(stdoutSim);
       try {
-        System.setProperty("basicauth", "harry:HarryIsUberCool");
```

Comment:
any chance that this pattern shows up anywhere else and or is suggested in docs?   Magic system variable?

Reply:
I have no idea, clearly it didn't work haha.